app/controllers/idv/capture_doc_controller.rb

```diff
@@ -25,13 +25,15 @@ def ensure_user_id_in_session
       process_result(result)
     end
 
-    def add_unsafe_eval_to_capture_steps
+    def override_document_capture_step_csp
       return unless current_step == 'document_capture'
 
-      # required to run wasm until wasm-eval is available
       SecureHeaders.append_content_security_policy_directives(
         request,
+        # required to run wasm until wasm-eval is available
         script_src: ['\'unsafe-eval\''],
+        # required for retrieving image dimensions from uploaded images
+        img_src: ['blob:'],
```

Comment:
We already [support](https://github.com/18F/identity-idp/blob/7e851cbd7d1c41f9e5059b3819eab19e13a2a00c/config/initializers/secure_headers.rb#L19-L21) `data:` URLs globally for images, which is arguably more permissive. We could just add this to the global configuration, though there's a certain appeal to knowing and limiting to exactly where we're expecting to need additional permissiveness.